### PR TITLE
One request to get all info to describe an application

### PR DESCRIPTION
**Pull request by janelletavares**
## Description of change

Include status and type in application description

Fixes https://github.com/meroxa/turbine-project/issues/129

## Type of change

- [ ]  New feature
- [ ]  Bug fix
- [x]  Refactor
- [ ]  Documentation

## How was this tested?

- [x]  Unit Tests
- [ ]  Tested in staging
- [ ]  Tested in minikube

## Demo

| before | after |
|--------|-------|
|
Screenshot from 2022-11-01 10-15-18
| 
Screenshot from 2022-11-02 11-56-01
|

### CODE REVIEW

**Review by ericcheatham — APPROVED** [review]

**Review by ericcheatham — DISMISSED** [review]
Noticed something when I re-ran the `m apps describe` command

Old 
```bash
 m apps describe meroxa-app-create-test
       UUID:   4c878073-0660-48fb-a882-019f6624136d
       Name:   meroxa-app-create-test
   Language:   python
    Git SHA:   e1dc745b270d76db1c67f219599658cd307445d8
 Created At:   2022-10-25 15:50:05 +0000 UTC
 Updated At:   2022-10-25 15:50:05 +0000 UTC
      State:   degraded
	Resources
	    webhook-dest (http-destination)
		 UUID:   1c9f95d2-b936-483f-9b3a-c4b49907b744
		 Type:   url
		State:   running
	    pg (jdbc-source)
		 UUID:   a21ff376-d716-4d0a-afbe-a7c1a6548132
		 Type:   postgres
		State:   failed

	Functions
	    blah-e1dc745b
		 UUID:   234be6e2-115f-4f73-b49b-64880f5a873f
		State:   running
```


new 
```bash
m apps describe meroxa-app-create-test
       UUID:   4c878073-0660-48fb-a882-019f6624136d
       Name:   meroxa-app-create-test
   Language:   python
    Git SHA:   e1dc745b270d76db1c67f219599658cd307445d8
 Created At:   2022-10-25 15:50:05 +0000 UTC
 Updated At:   2022-10-25 15:50:05 +0000 UTC
      State:   degraded
	Resources
	    webhook-dest (destination)
		 UUID:   1c9f95d2-b936-483f-9b3a-c4b49907b744
		 Type:
		State:
	    pg (source)
		 UUID:   a21ff376-d716-4d0a-afbe-a7c1a6548132
		 Type:
		State:

	Functions
	    blah-e1dc745b
		 UUID:   234be6e2-115f-4f73-b49b-64880f5a873f
		State:
```

The updated branch is missing details on  the resources and functions

**Review by ericcheatham — APPROVED** [review]

**Comment by janelletavares** [comment]
going to wait until the platform-api changes are merged and maybe until after the releases are done on Tuesday before merging this

**Pull request merged**
